server/meshmodel/kubearmor/relationships/hierarchical_inventory.json

```diff
@@ -0,0 +1,166 @@
+{
+    "apiVersion": "core.meshery.io/v1alpha1",
+    "kind": "Hierarchical",
+    "metadata": {
+      "description": "A hierarchical inventory relationship in which the configuration of (parent) component is patched with the configuration of other (child) component. Eg: The configuration of the EnvoyFilter (parent) component is patched with the configuration as received from WASMFilter (child) component."
+    },
+    "model": {
+      "name": "kubearmor",
+      "version": "*",
+      "displayName": "KubeArmor",
+      "category": {
+        "name": "Provisioning",
+        "metadata": null
+      },
+      "metadata": {}
+    },
+    "subType": "Inventory",
+    "evaluationQuery": "heirarchical_inventory_relationship",
+    "selectors": [
+      {
+        "allow": {
+          "from": [
+            {
+              "kind": "KubeArmorPolicy",
+              "model": "kubearmor",
+              "patch": {
+                "patchStrategy": "replace",
+                "mutatorRef": [
+                  [
+                    "name"
+                  ]
+                ],
+                "description": "KubeArmorPolicy defines rules governing system call behavior within Kubernetes Pods, ensuring security and compliance by controlling access to resources and actions. KubeArmorPolicy helps enforce fine-grained security measures tailored to specific application requirements and threat landscapes."
+              }
+            }
+          ],
+          "to": [
+            {
+              "kind": "Pod",
+              "model": "kubernetes",
+              "patch": {
+                "patchStrategy": "replace",
+                "mutatedRef": [
+                  [
+                    "annotations",
+                    "_", 
+                    "security.kubearmor.com/policy"
+                  ]
+                ],
+                "description": "Pods in Kubernetes are subjects to KubeArmorPolicy, enforcing security measures on the Pods' system calls and resource access, enhancing the overall security posture of containerized workloads."
```

Comment:
Update the description to be more oriented towards how Pod behaviour changes when annotated with KubeArmorPolicy .